pynest/nest/lib/hl_api_helper.py

```diff
@@ -443,39 +443,49 @@ def __show_help_in_modal_window(objname, hlptxt):
     display(Javascript(s.substitute(jstitle=objname, jstext=hlptxt)))
 
 
-def load_help(hlpobj, return_filename=False):
-    """Returns documentation of the object
+def get_help_filepath(hlpobj):
+    """Get file path of help object
 
     Parameters
     ----------
-    hlpobj : object
+    hlpobj : string
         Object to display help for
-    return_filename : bool
-        Option for returning filename
 
     Returns
     -------
     string:
-        The documentation of the object.
-    string:
-        Filename for the object
+        Filepath of the help object.
     """
 
     helpdir = os.path.join(os.environ['NEST_INSTALL_DIR'], "share", "doc",
```

Comment:
Sorry for not having caught this earlier, but please replace this line by
```Python
    helpdir = os.path.join(nest.sli_func("statusdict/prgdocdir ::"), "help")
```
This is IMO cleaner and does not rely on any environment variables being set.